src/main/java/core/basesyntax/model/FruitQuantity.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.model;
+
+public class FruitQuantity {
```

Comment:
Why do you need this model? Your storage isn't List of FruitQuantity, it's hashMap of Fruits and their respective quantities. IMO you won't need this class when you change the logic according to other comments